canonical/ubuntu.com
# [WD-8261] remove redirect from /embedded to /real-time

## Done

- Remove redirect from /embedded to /real-time
- Redirect /realtime-kernel to /real-time instead of to /embedded according to https://warthogs.atlassian.net/browse/WD-7729

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to https://ubuntu-com-13461.demos.haus/embedded and make sure a page for it opens without any redirects
- Navigate to https://ubuntu-com-13461.demos.haus/realtime-kernel and check if it redirects to /real-time

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8261

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
